metastore-grpc-proxy/metastore-grpc-proxy.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
```

Comment:
Please, add a test for this init action: https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/integration_tests

See BigTable init action for test example: https://github.com/GoogleCloudDataproc/initialization-actions/blob/master/bigtable/test_bigtable.py

Reply:
Done